crates/ruff/tests/lint.rs

```diff
@@ -780,6 +780,94 @@ fn each_toml_option_requires_a_new_flag_2() {
     ");
 }
 
+#[test]
+fn value_given_to_table_key_is_not_inline_table_1() {
+    // https://github.com/astral-sh/ruff/issues/13995
+    assert_cmd_snapshot!(Command::new(get_cargo_bin(BIN_NAME))
+        .args(STDIN_BASE_OPTIONS)
+        // spaces *also* can't be used to delimit different config overrides;
+        // you need a new --config flag for each override
```

Comment:
this comment is irrelevant to this test; I think it was just copied and pasted from the test immediately above

```suggestion
```